PhpUnit/DefinitionHasMethodCallConstraint.php

```diff
@@ -56,7 +67,7 @@ public function evaluate($other, $description = '', $returnResult = false)
     public function toString()
     {
         return sprintf(
-            'has a method call to "%s" with the given arguments',
+            'has a method call to "%s" with the given arguments on any or explicitly stated invocation order index.',
```

Comment:
It would be nice to make the warning explicit about whether or not a specific index has been checked (in fact, you might return a different warning instead of modifying the existing one).